### PR TITLE
FFWEB-1135 Migrate ff tag cloud to lit element

### PR DESCRIPTION
**Pull request by wdutkiewicz-obj**
Documentation adjusted to migrated `ff-tag-cloud` components. Minor fixes (formatting and typos) done.

### CODE REVIEW

File: markdown/3.x/en/api/ff-tag-cloud.api.md [hunk 1]
```diff
@@ -9,24 +9,16 @@ ___
...
 
 ### Events
 | Name | Description |
 | ---- | ----------- |
-| **entry-clicked** | Fired when a tag cloud entry is clicked and before the search for that entry is triggered. The event contains a reference to this element where you can set the property `ffPreventDefault` to skip its default action: Search for the clicked word. The event also contains an object with all the tag cloud entry information's: <br /> 1. query: The searched word <br /> 2. count: The number of searches for that word 3. params: The search parameters to execute a FACT-Finder search for that word. |
+| **entry-clicked** | Fired when a tag cloud entry is clicked and before the search for that entry is triggered. The event contains a reference to this element where you can set the property `ffPreventDefault` to skip its default action: Search for the clicked word. The event also contains an object with all the tag cloud entry information's: 1. query: The searched word. 2. count: The number of searches for that word. 3. params: The search parameters to execute a FACT-Finder search for that word. |
```
Comment (omikron-fp) [inline markdown/3.x/en/api/ff-tag-cloud.api.md]: Here is another potetial fix: `information's` -> `information`

Comment (omikron-fp) [reply markdown/3.x/en/api/ff-tag-cloud.api.md]: Already got fixed in the target branch.

File: markdown/3.x/en/ff-tag-cloud.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ Basic usage:
 <ff-tag-cloud min-font-size="10" max-font-size="40" word-count="50" unit="px"></ff-tag-cloud>
 ```
 **NOTE:**
-Changes to any property of the element, except `disable-auto`, will trigger a tag cloud request to FACT-Finder followed by a DOM update. The default action on a click of a tag cloud child element will trigger a FACT-Finder search for that word. You can prevent this behavior by setting the `ffPreventDefault` on the `entry-clicked` event.
+Changing `word-count` when `disable-auto` is not set will trigger a tag cloud request to FACT-Finder followed by a DOM update. The default action on a click of a tag cloud child element will trigger a FACT-Finder search for that word. You can prevent this behavior by setting the `ffPreventDefault` on the `entry-clicked` event.
```
Comment (omikron-fp) [inline markdown/3.x/en/ff-tag-cloud.md]:
```suggestion
Changing `word-count` when `disable-auto` is not set will trigger a tag cloud request to FACT-Finder followed by a DOM update. The default action of a click on a tag cloud child element will trigger a FACT-Finder search for that word. You can prevent this behavior by setting the `ffPreventDefault` on the `entry-clicked` event.
```

File: markdown/3.x/en/ff-tag-cloud.md [hunk 3]
```diff
@@ -10,7 +10,8 @@ Basic usage:
...
+`word-count` is the only property which triggers request to FACT-Finder when its value is changed.
```
Comment (omikron-fp) [inline markdown/3.x/en/ff-tag-cloud.md]:
```suggestion
`word-count` is the only property which triggers a request to FACT-Finder when its value is changed.
```

**Comment by wdutkiewicz-obj** [comment]
I rebased onto release/3.0 and added `tagCloud` topic to topic list. Please, check if I solved conflicts correctly.

**Pull request merged**